src/main/resources/gdpr/processing-basis.json

```diff
@@ -0,0 +1,8 @@
+[
```

Comment:
Each array element needs a unique ID so that we can attach logic to some of those in the front end